### DISCUSSION TITLE
Estimates in github issues

### DISCUSSION THREAD

**Discussion by safield** [community/community#6424] [Projects and Issues]
Just gave the new issues beta a spin.

I noticed that I can still not assigned estimates, and easily get a summary of work remaining on a certain milestone.

Is there any plans to add this ability?

**Comment by mversluys** [comment]
Numerical fields can be added, so the feature needed to support this would be for views to aggregate numerical fields?

**Reply by safield** [reply]
Ya that would be a simple improvement that would get a long ways. Also gantt chart would be nice.

**Reply by tijmenvangurp** [reply]
Would also be nice to be able to see the total amount for an estimate per person per sprint, this could of course be achieved by having the total of the column, and simply filter per person/group by iteration.

**Reply by belluede** [reply]
yes please! i want to get rid of zenhub.
